tests/communications/test_notification_preferences.py

```diff
@@ -0,0 +1,156 @@
+from django.urls import reverse
+from rest_framework import status
+from rest_framework.test import APITestCase
+from django.contrib.auth import get_user_model
+from rest_framework.authtoken.models import Token
+from communications.models import NotificationType, UserNotificationPreference, UserNotificationTypePreference
+
+import logging
+logging.disable(logging.CRITICAL)
```

Comment:
Disabling logging globally with logging.disable(logging.CRITICAL) will suppress all critical logs during the test run, which may hide important errors or warnings. Consider using Django's override_settings or patching logging only for specific noisy loggers.